inc/namespace.php

```diff
@@ -255,3 +260,12 @@ function get_elasticsearch_url() : string {
 	$host = sprintf( '%s://%s:%d', ELASTICSEARCH_PORT === 443 ? 'https' : 'http', ELASTICSEARCH_HOST, ELASTICSEARCH_PORT );
 	return $host;
 }
+
+/**
+ * When WordPress is installed via WP-CLI, run the ElasticPress setup
```

Comment:
Full stops at the end of comments
```suggestion
 * When WordPress is installed via WP-CLI, run the ElasticPress setup.
```